src/ansys/dpf/core/server_context.py

```diff
@@ -38,6 +40,111 @@ def same_licensing_context(first, second):
         return True
 
 
+class LicenseContextManager:
+    """Can be optionally used to pre checkout a licence before using licensed DPF Operators.
+    Improves performances if many Operators requiring licensing are used afterwards.
+    It can also be used to force checkout before running a script when few
+    ANSYS license increments are available.
+    Check in the license when the object is deleted.
```

Comment:
For line 48, my assumption is that the check in is automatic when the object is deleted. If you have to manually check in the license after deleting the object, keep line 48 as originally written.

Reply:
that's correct @JennaPaikowsky, thanks